models/aeif_cond_beta_multisynapse.h

```diff
@@ -0,0 +1,418 @@
+/*
+ *  aeif_cond_beta_multisynapse.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef AEIF_COND_BETA_MULTISYNAPSE_H
+#define AEIF_COND_BETA_MULTISYNAPSE_H
+
+// Generated includes:
+#include "config.h"
+
+#ifdef HAVE_GSL
+
+// External includes:
+#include <gsl/gsl_errno.h>
+#include <gsl/gsl_matrix.h>
+#include <gsl/gsl_odeiv.h>
+
+// Includes from nestkernel:
+#include "archiving_node.h"
+#include "connection.h"
+#include "event.h"
+#include "nest_types.h"
+#include "ring_buffer.h"
+#include "universal_data_logger.h"
+
+/* BeginDocumentation
+ Name: aeif_cond_beta_multisynapse - Conductance based exponential
+                                      integrate-and-fire neuron model according
+                                      to Brette and Gerstner (2005) with
+                                      multiple synaptic rise time and decay
+                                      time constants, and synaptic conductance
+                                      modeled by a beta function.
+
+ Description:
+
+ aeif_cond_beta_multisynapse is an extension of
+ aeif_cond_alpha_multisynapse.  It allows an arbitrary number of synaptic
+ rise time and decay time constant. Synaptic conductance is modeled by a
+ beta function, as described by A. Roth and M.C.W. van Rossum
+ in Computational Modeling Methods for Neuroscientists, MIT Press 2013,
+ Chapter 6
+
+ The time constants are supplied by two arrays taus_rise and taus_decay for
+ the synaptic rise time and decay time, respectively.  Port numbers
+ are then automatically assigned and there range is from 1 to n.  (n
+ being the index of the last element of the tau_ex and tau_in
+ arrays).
+ During connection, the ports are selected with the property "receptor_type".
+
+ Examples:
+ % PyNEST example, of how to assign synaptic rise time and decay time
+ % to a receptor type.
+
+ nest.SetDefaults('aeif_cond_beta_multisynapse', {'HMIN':0.001})
+ nest.SetDefaults('aeif_cond_beta_multisynapse', {'MAXERR':1e-10})
```

Comment:
I think these two parameters have been replaced by tolerance parameters for the GSL solver?